### DISCUSSION TITLE
How to save the value of Street Profile into each building footprint.

### DISCUSSION THREAD

**Discussion by cdkang** [pysal/momepy#445] [Q&A]
Hello, 

I appreciate your support as your comments made me a lot of progress in applying momepy in my study. 

Here is my new questions. 

Momepy saves the value of Street Profile function into street file. I am wondering how to save the value of Street Profile into each building footprint.  Just do we assume the value of street profile in each building unit is the nearest value in street maps from the building units? 

Sincerely,

**Comment by martinfleis** [comment]
At the moment, we just link a building to a nearest street segment. If you wanted to have differences per building along the street segment, you'd have to split street segments to smaller parts.

**Reply by cdkang** [reply]
Thank you for your answer. By the way, is there any function to split street segments by nearerst buildings in ArcGIS or momepy?

**Reply by martinfleis** [reply]
No idea about ArcGIS. Nothing in momepy.

**Comment by cdkang** [comment]
Thank you for your reply.
